pkg/csi/service/wcp/controller_helper.go

```diff
@@ -526,8 +526,12 @@ func (c *controller) GetVolumeToHostMapping(ctx context.Context) (map[string]str
 		vmMoID := info.Reference().Value
 
 		host := info.Runtime.Host
-		vmMoIDToHostMoID[vmMoID] = host.Reference().Value
-
+		if host != nil {
```

Comment:
nit: Use `info.Runtime.Host` to check != nil

Reply:
done